### PR TITLE
Upgrade pgtrigger and test against Django 3.1

### PR DESCRIPTION
**Pull request by wesleykendall**
Uses the latest version of django-pgtrigger. Also tests against Django 3.1
and fixes a few bugs related to internal changes in the Django codebase.

Type: feature

@tomage This tests against Django 3.1 and also upgrades pghistory to use the latest django-pgtrigger. pghistory was susceptible to a previous bug in django-pgtrigger that might have caused underlying triggers to overwrite each other's code.

It would be good if https://github.com/jyveapp/public-django-app-template/pull/1 was merged first so I can go ahead and update this to reference the latest template

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -34,14 +34,14 @@ documentation = "https://django-pghistory.readthedocs.io"
...
-django-pgtrigger = "^1.0.0"
+django-pgconnection = ">=1.0.0"
+django-pgtrigger = ">=2.1.0"
 
 [tool.poetry.dev-dependencies]
 black = "=19.10b0"
 boto3 = "=1.9.243"
 dj-database-url = "^0.5.0"
-django-dynamic-fixture = "^3.1.0"
+django-dynamic-fixture-dj31 = "=3.1.1"
```
Comment (wesleykendall) [inline pyproject.toml:44]: There's been an oustanding pr for ddf to have 3.1 compatibility for some time now. I went ahead and deployed a fork as a temporary fix and will revert this back once the author has released the fix

**Comment by wesleykendall** [comment]
Just updated to reference the latest template

**Review by tomage — APPROVED** [review]
Good catch!

**Review by tomage — APPROVED** [review]
Still looks good to me!

Clever thing with the `JSONField`. (and happy to hear Django is making it part of the "official" set of fields they offer in "core" Django.

**Comment by wesleykendall** [comment]
@tomage thanks!

**Pull request merged**